tests/integ/test_kmeans.py

```diff
@@ -75,6 +75,11 @@ def test_kmeans(sagemaker_session):
             assert record.label["closest_cluster"] is not None
             assert record.label["distance_to_cluster"] is not None
 
+    predictor.delete_model()
```

Comment:
is there an integ test that successfully deletes the model?

Reply:
The existing integ tests only delete endpoint, except for test_transformer which deletes model.

Reply:
Disregard this comment. This test deletes the model. You're doing a describe to confirm it.